efficientdet/det_model_fn.py

```diff
@@ -512,7 +514,11 @@ def _model_outputs(inputs):
       raise ValueError('optimizers should be adam or sgd')
     if params['use_tpu']:
       optimizer = tf.tpu.CrossShardOptimizer(optimizer)
-
```

Comment:
Should this be "elif params['use_horovod']". I am not sure whether horovod is compatible to TPU optimizer or not.